test/Generator/TypeGeneratorTest.php

```diff
@@ -241,6 +241,9 @@ public function invalidTypeProvider()
             ['\\iterable'],
             ['\\Iterable'],
             ['\\ITERABLE'],
+            ['\\object'],
+            ['\\Object'],
+            ['\\OBJECT'],
```

Comment:
Here you added these to `invalidTypeProvider`, don't we need remove 'object' from `validTypeProvider`?